Makefile

```
@@ -633,6 +633,17 @@ endif
 	git commit -m "Adding $(MLRUN_VERSION) tag contents" --allow-empty; \
 	git push origin $$BRANCH_NAME
 
+.PHONY: api-backward-compatibility
+api-backward-compatibility: ## Check for api backward compatibility breakage
+ifndef BASE_PATH
+	$(error BASE_PATH is undefined)
+endif
+	mkdir base && mkdir head ;\
+	export OPENAPI_JSON_TARGET_PATH=$(PWD)/base &&\
```

```suggestion
	export MLRUN_OPENAPI_JSON_TARGET_PATH=$(PWD)/base &&\
```
rename this env var in other places as well